src/SFA.DAS.ApplyService.Web/Controllers/ApplicationController.cs

```diff
@@ -304,92 +304,30 @@ private void ProcessPageVmQuestionsForStandardName(List<QuestionViewModel> pageV
-
-                if (applyValidationRulesOnSaveAndContinue)
-                {
-                    if (answers.Any(a => inputEnteredRegex.IsMatch(a.Value)))
-                    {
-                        var invalidSaveResult =
-                            await SaveAnswers(applicationId, sequenceId, sectionId, pageId, redirectAction);
-
-                        if (!ModelState.IsValid) return invalidSaveResult;
-                    }
-                }
-            }
-
-            // If we got to here then all is well
-            if (redirectAction == "Feedback")
-            {
-                return RedirectToAction("Feedback", new { applicationId });
-            }
-            else
-            {
-                var thisPage = await _apiClient.GetPage(applicationId, sequenceId, sectionId, pageId, User.GetUserId());
-                if (thisPage.PageOfAnswers.Any())
-                {
-                    var next = thisPage.Next.FirstOrDefault();
-                    if (next == null)
-                    {
-                        return RedirectToAction("Section", "Application", new { applicationId, sectionId = thisPage.SectionId });
-                    }
-
-                    if (next.Action == "NextPage")
-                    {
-                        return RedirectToAction("Page", new { applicationId, sequenceId = thisPage.SequenceId, sectionId = thisPage.SectionId, pageId = next.ReturnId, redirectaction = redirectAction });
-                    }
-
-                    return next.Action == "ReturnToSection"
-                        ? RedirectToAction("Section", "Application", new { applicationId, sectionId = next.ReturnId })
-                        : RedirectToAction("Sequence", "Application", new { applicationId });
-                }
-
-                return RedirectToAction("Page", new { applicationId, sequenceId = thisPage.SequenceId, sectionId = thisPage.SectionId, pageId = thisPage.PageId, redirectaction = redirectAction });
-            }
-        }
-
-        private async Task<bool> CheckIfValidationRequiredOnSaveAndContinue(Guid applicationId, int sequenceId, int sectionId,
+        /// <summary>
+        /// Checks if answers must be validated
+        /// </summary>
+        /// <param name="applicationId">The application identifier.</param>
+        /// <param name="sequenceId">The sequence identifier.</param>
+        /// <param name="sectionId">The section identifier.</param>
+        /// <param name="pageId">The page identifier.</param>
+        /// <param name="answers">The answers.</param>
+        /// <param name="inputEnteredRegex">The input entered regex which determines if an answer has been entered.</param>
+        /// <returns>True when any answer has been entered or no previous answers have been entered for a multi answer page; False otherwise.</returns>
```

Comment:
The majority of this summary doesn't tell us any more than the method signature and is just noise.

Reply:
I'll rework that